### ISSUE TITLE
VZD - Missing records

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/atd-data-tech#4875]
```
Case ID      Crash Date   Crash Time    Status
203640632  2020-12-29  12:40:00     APPROVED  ACCEPTED BY TXDOT  system  12/29/2020 3:38:00 PM
203651018  2020-12-30  18:20:00      APPROVED  ACCEPTED BY TXDOT  system  12/31/2020 11:54:00 AM
203641379  2020-12-29  21:34:00      APPROVED  ACCEPTED BY TXDOT  system  12/31/2020 10:39:00 AM
203530298  2020-12-18  06:18:00      APPROVED  ACCEPTED BY TXDOT  system  12/29/2020 12:35:00 PM
203480044  2020-12-13  00:34:00      APPROVED  ACCEPTED BY TXDOT  system  1/4/2021 10:59:00 AM    (This one is fairly new, so I assume it'll be in our system in the coming days)
202160250  2020-08-03  05:50:00      APPROVED  ACCEPTED BY TXDOT  system  9/11/2020 4:21:00 PM 
```

1st thing to do to fix this is reimport the backlog of missing days since our last manual insert.

**Comment by mateoclarke** [comment]
After working on fixing breaking changes caused by the CRIS 21.0 update on 12/31, we've been able to reimport all Dec 2020 records and it covers missing data for most of these.

The one record still missing from the import is `202160250`. This record does appear in the CRIS Search platform so we should have imported it. Sergio and I have proposed running a bulk import for all 2020 records to try to catch any records that might be missing like this one.
